tasks/serve/index.ts

```diff
@@ -79,15 +80,20 @@ export class clitask {
         //we need to force pull the docker image, in case the tag was local already (but old content)
         let p = tl.tool('docker').arg('pull').arg(recipe).exec()        
         p.then(()=>{
+            
+            //Get the Docker Options    
+            dockerOptions = tl.getInput('dockerOptions') 
+
             let tool = tl.tool('docker')
 
             let args = tool.arg('run').arg('--rm').arg('-t')
                 .arg('--env-file=' + envFile)
                 .arg(`-v=${process.env.BAKE_VARIABLES}:/app/bake/.env:Z`)                
-            //if (dockerindocker === true)                
-            //    { 
-            //        args= args.arg(`-v=/var/run/docker.sock:/var/run/docker.sock`)
-            //    }
+            
+                //Add Docker Options, if exists
+            if (dockerOptions) {
+                    args= args.arg(dockerOptions)            
```

Comment:
this only supports a single option. so dockerOptions an not contain multiple space-separated options. The entire string will get parsed as a single argument to docker.